site/docs/install-os-x.md

```diff
@@ -141,3 +141,11 @@ following command:
 ```shell
 $ brew upgrade bazel
 ```
+
+To install older bazel versions
+
+```
+brew tap bazelbuild/tap
```

Comment:
> Do you know who maintains the homebrew integration? I have no idea myself.

This command clones the https://github.com/bazelbuild/homebrew-tap repo as documented [here](https://docs.brew.sh/Taps#the-brew-tap-command). This repo is in the same org as this one so I assume it's owned by the official bazel team?